server/sonar-web/src/main/js/main/nav/global/search-view.js

```diff
@@ -207,7 +207,7 @@ export default Marionette.LayoutView.extend({
           collection.push(_.extend(item, {
             q: domain.q,
             extra: index === 0 ? domain.name : null,
-            url: '/dashboard/index?id=' + encodeURIComponent(item.key) + window.dashboardParameters(true)
+            url: window.baseUrl + '/dashboard/index?id=' + encodeURIComponent(item.key) + window.dashboardParameters(true)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 122 characters long line (which is greater than 120 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3ALineLength)